src/vs/workbench/contrib/notebook/browser/notebookChatEditController.ts

```diff
@@ -457,30 +464,51 @@ class NotebookModelSynchronizer extends Disposable {
 	public readonly onDidUpdateNotebookModel = this._onDidUpdateNotebookModel.event;
 	private readonly _onDidRevert = this._register(new Emitter<boolean>());
 	public readonly onDidRevert = this._onDidRevert.event;
+	private readonly _onDidAccept = this._register(new Emitter<void>());
+	public readonly onDidAccept = this._onDidAccept.event;
 	private snapshot?: { bytes: VSBuffer; dirty: boolean };
 	private isEditFromUs: boolean = false;
 	constructor(
 		private readonly notebookEditor: INotebookEditor,
 		public readonly entry: IModifiedFileEntry,
 		private readonly viewType: string,
 		@INotebookService private readonly notebookService: INotebookService,
+		@IChatService chatService: IChatService,
 		@INotebookLoggingService private readonly logService: INotebookLoggingService,
 		@IConfigurationService private readonly configurationService: IConfigurationService,
 		@INotebookEditorWorkerService private readonly notebookEditorWorkerService: INotebookEditorWorkerService,
 		@INotebookEditorModelResolverService private readonly notebookModelResolverService: INotebookEditorModelResolverService,
 	) {
 		super();
+
+		this._register(chatService.onDidPerformUserAction(async e => {
+			if (e.action.kind === 'chatEditingSessionAction' && !e.action.hasRemainingEdits && e.action.uri === entry.modifiedURI) {
```

Comment:
You should not compare uri-objects by identity. Use the `resources#isEqual` utility or their string representation.

Reply:
Oh bugger, yes you are right.